source/adapters/cuda/program.cpp

```diff
@@ -188,6 +188,26 @@ urProgramCompile(ur_context_handle_t hContext, ur_program_handle_t hProgram,
   return urProgramBuild(hContext, hProgram, pOptions);
 }
 
+UR_APIEXPORT ur_result_t UR_APICALL urProgramCompileExp(
+    ur_context_handle_t hContext, ur_program_handle_t hProgram,
+    uint32_t numDevices, ur_device_handle_t *phDevices, const char *pOptions) {
+  std::ignore = numDevices;
+  std::ignore = phDevices;
+  return urProgramBuild(hContext, hProgram, pOptions);
```

Comment:
should be `urProgramCompile`, right?

Reply:
hip/cuda plugins call urProgramBuild in urProgramCompile, this was calling directly to build since compile already does that.

To be cleaner I have just setup the function definitions for Exp functions in hip/cuda to be unsupported since the pi2ur does a fallback anyway.